recipes/gottcha2/2.1.4_beta/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set name = "GOTTCHA2" %}
+{% set version = "2.1.4_beta" %}
```

Comment:
Do we really need the beta version? Can we wait for a stable version.

Reply:
@poeli would you as the upstream author consider to package GOTTCHA2 using a setup.py file? Another improvement would be to include a download function to retrieve the databases automatically.

Reply:
@bgruening The version is quite stable -- I'm comfortable to remove the "beta".
@druvus Thanks for the suggestions. I will make one.

Reply:
I would keep this PR open until @druvus have requested/added a non-beta version upstream with the database download function